web/src/routes/(user)/map/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -18,6 +18,22 @@
   import type { PageData } from './$types';
   import { handlePromiseError } from '$lib/utils';
   import { navigate } from '$lib/utils/navigation';
+  import { AssetStore } from '$lib/stores/assets.store';
+  import { AssetInteraction } from '$lib/stores/asset-interaction.svelte';
+  import AssetGrid from '$lib/components/photos-page/asset-grid.svelte';
+  import { page } from '$app/stores';
+  import { t } from 'svelte-i18n';
+  import { mdiArrowDown, mdiArrowUp } from '@mdi/js';
+  import Button from '$lib/components/elements/buttons/button.svelte';
+  import Icon from '$lib/components/elements/icon.svelte';
+  import maplibregl from 'maplibre-gl';
```

Comment:
That this bleeds out into other pages is kind of unfortunate and probably indicates that the architecture isn't ideal.